DashDocs/src/utils/macro.jl

```diff
@@ -0,0 +1,52 @@
+macro eval_toplevel()
```

Comment:
Why is this a macro? This can be a function.

Reply:
This is an eval of expressions block  that should be executed at the top level of DashDocs. It seems to me that it is more convenient to do this with a macro than with an direct eval call. Perhaps the name is not very successful